core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator.go

```diff
@@ -302,6 +317,43 @@ func (c *coordinator) getBlocks(
 	return
 }
 
+// getBlockhashesMapping returns the blockhashes within a range.
+func (c *coordinator) getBlockhashesMapping(
```

Comment:
Looks like `getBlocks` can just use `getBlockhashesMapping`? A lot of the code is similar.